types/index.d.ts

```diff
@@ -0,0 +1,4 @@
+export function HomebaseProvider(props?: object): object;
+export function useTransact(): Array<Array<object>>;
+export function useEntity(lookup: object | string | number): Array<any>;
```

Comment:
We should add an Entity type that has a get function that takes 1 or more strings.
Then useEntity can return a Tuple where the first element is an Entity or null.
The lookup should only be `object | number`